<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QWidget" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_login_logo">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(:/img/im_logo.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_login_id">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_login_password">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>234</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_login_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>176</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>id</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_login_password">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login_lose_password">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>lose password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login_register">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>register</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>461</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
